thirdeye/thirdeye-pinot/src/test/java/Test.java

```diff
@@ -0,0 +1,87 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.linkedin.thirdeye.constant.MetricAggFunction;
+import com.linkedin.thirdeye.detector.api.AnomalyFunctionSpec;
+
+public class Test {
```

Comment:
This was a test class for a completely different thing. Sorry forgot to erase if before adding. Removed.